pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -373,7 +352,7 @@ func (cr *PerconaServerMongoDB) CheckNSetDefaults(platform version.Platform, log
 
 		if replset.ReadinessProbe.TCPSocket == nil {
 			replset.ReadinessProbe.TCPSocket = &corev1.TCPSocketAction{
-				Port: intstr.FromInt(int(cr.Spec.Mongod.Net.Port)),
+				Port: intstr.FromInt(int(DefaultMongodPort)),
```

Comment:
Do I understand correctly that with this we will loose the possibility to change the mongo port and only default 27017 will be used?

Reply:
@tplavcic yes, you are right

Reply:
👍